docs/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@lodestar/docs",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
```

Comment:
How do you feel about adding a "clean" command that removes the artifacts from `prepare-docs` and runs the `clear` below?

Reply:
Not sure about removing the artifacts from the script, sounds like a good way to shoot ourselves in the foot.
Actually I'm hoping we can do smarter things now that we use a js based framework, so would rather left this out of this PR.